master/docs/Makefile

```diff
@@ -10,8 +10,6 @@ TAR_TRANSFORM := $(if $(filter bsdtar,$(TAR_VERSION)),-s /^html/$(VERSION)/,--tr
 docs.tgz: clean html singlehtml images-png
 	sed -e 's!href="index.html#!href="#!g' < _build/singlehtml/index.html > _build/html/full.html
 	tar -C _build $(TAR_TRANSFORM) -zcf $@ html
-pipdeps:
```

Comment:
In short, Makefiles describes set of targets (usually files), their dependencies and how to build them.

In this example:

```
.PHONY: tutorial manual pipdeps

pipdeps:                                                                                                                                          
        pip install sphinxcontrib-blockdiag 'docutils>=0.8'

html: conf.py pipdeps
        $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

two targets are defined: `html` and `pipdeps`.

To build them you can run `make html` and `make pipdeps`

To build `pipdeps` target make will issue commands that are specified below target: `pip install sphinxcontrib-blockdiag 'docutils>=0.8'`.

`html` target has two dependencies: `conf.py` file and `pipdeps` target.
They will be built first before invoking commands to build `html` target.

By default targets are files. `.PHONY` is used to specify that target is not a file (it's strange to me that `html` is not a phony target here).

So you need to remove from other targets `pipdeps` as dependency:

```
-html: conf.py pipdeps
+html: conf.py
```